go.sum

```diff
@@ -270,15 +270,11 @@ github.com/gophercloud/gophercloud v0.3.0/go.mod h1:vxM41WHh5uqHVBMZHzuwNOHh8XEo
 github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1 h1:EGx4pi6eqNxGaHF6qqu48+N2wcFQ5qg5FXgOdqsJ5d8=
 github.com/gopherjs/gopherjs v0.0.0-20181017120253-0766667cb4d1/go.mod h1:wJfORRmW1u3UXTncJ5qlYoELFm8eSnnEO6hX4iZ3EWY=
 github.com/gorilla/context v1.1.1/go.mod h1:kBGZzfjB9CEq2AlWe17Uuf7NDRt0dE0s8S51q0aT7Yg=
-github.com/gorilla/handlers v1.4.2 h1:0QniY0USkHQ1RGCLfKxeNHK9bkDHGRYGNDFBCS+YARg=
```

Comment:
can you merge with latest and re-run mod tidy to fix any weirdness from the set of renovate bot merges?

Reply:
I did, renovate bot seems never run `go mod tidy` :(

Reply:
https://github.com/pomerium/pomerium/blob/3d7206dc1e61864c9ed94c45abdf1c6f20440b06/renovate.json#L12

So weird.

Reply:
I think that should be a bug, or they run `go mod tidy` with some special option, causes the bot never remove old dependencies in `go.sum`.